File: JavaScript/1038 - Lanche.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8').split(' ').map(Number);
var codigo = input[0];
var quantidade = input[1];

var precos = {1: 4.00, 2: 4.50, 3: 5.00, 4: 2.00, 5: 1.50};
var total = precos[codigo] * quantidade;

console.log(`Total: R$ ${total.toFixed(2)}`);

File: Clojure/1038 - Lanche.clj
(def codigo (read))
(def quantidade (read))

(cond
    (== codigo 1) (printf "Total: R$ %.2f\n" (* 4.00 quantidade))
    (== codigo 2) (printf "Total: R$ %.2f\n" (* 4.50 quantidade))
    (== codigo 3) (printf "Total: R$ %.2f\n" (* 5.00 quantidade))
    (== codigo 4) (printf "Total: R$ %.2f\n" (* 2.00 quantidade))
    (== codigo 5) (printf "Total: R$ %.2f\n" (* 1.50 quantidade))
    )